src/Model/Subscription.php

```diff
@@ -66,18 +68,23 @@ public function toArray(): array
         $price = $this->getPrice();
         $payment = $this->getPaymentMethod();
 
-        return array_filter([
-            'id' => $this->getId(),
-            'startDate' => $this->getStartDate(),
-            'price' => empty($price) ? null : $price->toArray(),
-            'status' => $this->getStatus(),
-            'statusHistory' => $this->getStatusHistory(),
-            'daysPastDue' => $this->getDaysPastDue(),
-            'customer' => $this->getCustomer()->toArray(),
-            'payment' => empty($payment) ? null : $payment->toArray(),
-            'plan' => $this->plan->toArray(),
-            'nextBillingPeriodAmount' => $this->getNextBillingPeriodAmount(),
-        ]);
+        return array_merge(
+            array_filter([
+                'id' => $this->getId(),
+                'startDate' => $this->getStartDate(),
+                'price' => empty($price) ? null : $price->toArray(),
+                'status' => $this->getStatus(),
+                'statusHistory' => $this->getStatusHistory(),
+                'daysPastDue' => $this->getDaysPastDue(),
+                'customer' => $this->getCustomer()->toArray(),
+                'payment' => empty($payment) ? null : $payment->toArray(),
+                'plan' => $this->plan->toArray(),
+                'nextBillingPeriodAmount' => $this->getNextBillingPeriodAmount(),
+            ]),
+            [
+                'isProrated' => $this->isProrated(),
```

Comment:
Is this because the `array_filter` was removing the `false` value?

Reply:
@brianium Yes.